<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>691</width>
      <height>421</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Stand Configuration</string>
     </attribute>
     <widget class="QLabel" name="statusLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>350</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Status: </string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>250</y>
        <width>82</width>
        <height>58</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>52</x>
        <y>71</y>
        <width>397</width>
        <height>95</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="djiInfo">
           <property name="text">
            <string>info</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Enable DJI</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="djiBox">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="ardroneInfo">
           <property name="text">
            <string>info</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Enable AR Drone</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ardroneBox">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Automatic Takedown</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="ardroneAuto">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="mavlinkInfo">
           <property name="text">
            <string>info</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Enable MavLink</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="mavlinkBox">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Automatic Takedown</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="mavlinkAuto">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <zorder>startButton</zorder>
     <zorder>stopButton</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_4</zorder>
     <zorder>djiBox</zorder>
     <zorder>ardroneBox</zorder>
     <zorder>mavlinkBox</zorder>
     <zorder>label_4</zorder>
     <zorder>label_2</zorder>
     <zorder>djiInfo</zorder>
     <zorder>ardroneInfo</zorder>
     <zorder>mavlinkInfo</zorder>
     <zorder>statusLabel</zorder>
     <zorder>label_3</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
     <zorder>ardroneAuto</zorder>
     <zorder>djiAuto</zorder>
     <zorder>mavlinkAuto</zorder>
     <zorder>label_6</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
